frontend/Dockerfile

```
@@ -2,14 +2,20 @@ FROM python:3.13-slim
 
 WORKDIR /app
 
-RUN apt-get update && apt-get upgrade -y
+RUN apt-get update && apt-get install -y \
+    gcc \
+    libportaudio2 \
+    libportaudiocpp0 \
+    portaudio19-dev
 
 COPY requirements.txt .
 
 RUN pip3 install --upgrade pip
 RUN pip3 install -r requirements.txt
 
 COPY main.py ./
+COPY agent.py ./
+COPY *.json ./
```

Sth like this also should work. It's more compact and easier to read for a few files.

```suggestion
COPY main.py agent.py *.json ./
```